acoular/configuration.py

```diff
@@ -80,61 +80,61 @@ def __init__(self):
         HasStrictTraits.__init__(self)
         self._assert_h5library()
 
-    #: Flag that globally defines caching behaviour of Acoular classes
-    #: defaults to 'individual'.
-    #:
-    #: * 'individual': Acoular classes handle caching behavior individually.
-    #: * 'all': Acoular classes cache everything and read from cache if possible.
-    #: * 'none': Acoular classes do not cache results. Cachefiles are not created.
-    #: * 'readonly': Acoular classes do not actively cache, but read from cache if existing.
-    #: * 'overwrite': Acoular classes replace existing cachefile content with new data.
+    # Flag that globally defines caching behaviour of Acoular classes
+    # defaults to 'individual'.
+    #
+    # * 'individual': Acoular classes handle caching behavior individually.
+    # * 'all': Acoular classes cache everything and read from cache if possible.
+    # * 'none': Acoular classes do not cache results. Cachefiles are not created.
+    # * 'readonly': Acoular classes do not actively cache, but read from cache if existing.
+    # * 'overwrite': Acoular classes replace existing cachefile content with new data.
     global_caching = Property()
 
     _global_caching = Trait('individual', 'all', 'none', 'readonly', 'overwrite')
 
-    #: Flag that globally defines package used to read and write .h5 files
```

Comment:
I remember that the `#:` was needed at least at some time in the past to make traits documenting work. This may have changed. To be sure, please build the documentation and check if all traits are still documented properly when just `#` is used. Without the traits documenter extension traits are simple understood as class variables and the documentation lacks the information.

Reply:
Meanwhile I have checked that the '#:' is indeed needed. The documentation build with this PR is incomplete. See https://github.com/acoular/acoular/actions/runs/11859469449/artifacts/2193862356 and compare with https://www.acoular.org/index.html .  All traits documentation is missing.

Reply:
oh no :sweat: :hole: I didn't know about this functionality. Since the blame always showed Tom for these lines, I figured that it was just his style/formatter that did this.